apps/homepage/src/app/[lang]/page.tsx

```diff
@@ -1,13 +1,87 @@
 'use client'
 import styles from './home.module.css'
-import { useLocalization } from '../../lib/context/loc-context'
+import { motion, AnimatePresence } from 'framer-motion';
+import { useEffect, useState } from 'react';
+import { UiViewport } from '@uireact/foundation';
+import { renderDynamicPhrase } from './utils';
+
+const phraseData = {
+  phrase: "We {keyWords} the future while being {keyDescriptions}",
+  keyWords: ['{engineer}', '{create}', '{code}'],
+  keyDescriptions: ['{humans}', '{developers}', '{artists}']
+}
 
 export default function AppPage() {
-  const loc = useLocalization();
+  const { getPhrase, next } = renderDynamicPhrase(phraseData);
+  const [phrase, setPhrase] = useState(getPhrase());
+
+  useEffect(() => {
+    const interval = setInterval(() => {
+      setPhrase(next());
+    }, 3000);
```

Comment:
I would make this configurable through props in your UI component so anybody using this component could configure their own internal time